src/Tasks/Microsoft.NET.Build.Tasks/LockFileExtensions.cs

```diff
@@ -76,6 +83,38 @@ public static LockFileTargetLibrary GetLibrary(this LockFileTarget lockFileTarge
 
             return projectDeps;
         }
+        
+        public static HashSet<string> GetIntersetction(
+            IDictionary<string, LockFileTargetLibrary> coll1,
```

Comment:
It is standard coding in C# to not use abbreviations or contractions in variable names.

`coll1`
`iter`